src/components/bar.cpp

```diff
@@ -458,7 +458,16 @@ void bar::restack_window() {
 
   auto restacked = false;
 
-  if (wm_restack == "bspwm") {
+  if (wm_restack == "generic") {
+    auto children = m_connection.query_tree(m_connection.screen()->root).children();
+    auto lowest = *children.begin();
```

Comment:
This may be a bit pedantic, but could chidlren ever be empty?

In theory, this shouldn't happen because polybar is a child of the root, so the root must have at least one child.
I'm just wondering if there can't be some kind of weirdness.